<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>0</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>SHA</th>
      <td>0b460e73926eb107001c95c0dfeae2362b4251b0</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Precautions are Needed for COVID-19 Patients with Coinfection of Common Respiratory Pathogens</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Quan-Sheng Xing</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Qingdao University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101/2020.02.29.20027698</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>With the ongoing outbreak of Coronavirus Disease 2019 , infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>1d2e66528cfb3f754157404a225d551f6cdab1ae</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Epidemiology of seasonal coronaviruses: Establishing the context for COVID-19 emergence</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Sema Nickbakhsh</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of Glasgow</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101/2020.03.</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Public health preparedness for coronavirus disease 2019 is challenging in the absence of setting-specific epidemiological data. Here we describe the epidemiology of seasonal human coronaviruses (sCoVs) and other cocirculating viruses in the West of Scotland, UK. We analyzed routine diagnostic data for &gt;70,000 episodes of respiratory illness tested molecularly for multiple respiratory viruses between 2005 and 2017. Statistical associations with patient age and sex differed between CoV-229E, CoV-OC43 and CoV-NL63. Furthermore, the timing and magnitude of sCoV outbreaks did not occur concurrently and coinfections were not reported. With respect to other cocirculating respiratory viruses, we found evidence of positive, rather than negative, interactions with sCoVs. These findings highlight the importance of considering cocirculating viruses in the differential diagnosis of COVID-19. Further work is needed to establish the occurrence/degree of cross-protective immunity conferred across sCoVs and with COVID-19, as well as the role of viral coinfection in COVID-19 disease severity.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>2e3b61da2cfd48b52834db5e8d624fb9b0d21fd5</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Pulmonary pathology of severe acute respiratory syndrome in Toronto</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>David Hwang</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The severe acute respiratory syndrome (SARS) pandemic in Toronto resulted in a large number of autopsies on its victims. We describe the pulmonary pathology of patients who died in the 2003 Toronto outbreak. Autopsy material from the lungs of 20 patients who died between March and July 2003 were characterized by histology, molecular biology, and immunohistochemistry for cytokeratins, thyroid transcription factor-1, CD68, Epstein-Barr virus, cytomegalovirus, and human herpes simplex viruses. Matched controls were obtained from patients who died of other causes over the same interval. The mean duration of illness was 27 days (range 5-108 days). Post-mortem lung tissues from 19 of 20 patients with probable SARS were positive for SARS-associated coronavirus by RT-PCR. Histologically, all patients showed varying degrees of exudative and proliferative phase acute lung injury, evidenced in conventional and immunohistochemical stains by edema, inflammatory infiltrate, pneumocyte hyperplasia, fibrinous exudates, and organization. Eight of 20 patients showed predominantly a diffuse alveolar damage pattern of acute lung injury, six showed predominantly an acute fibrinous and organizing pneumonia pattern, and the remainder showed an admixture of the two patterns. Squamous metaplasia and scattered multinucleate giant cells were present in most cases. Vascular fibrin thrombi were a common finding and were often associated with pulmonary infarcts. Special stains demonstrated vascular endothelial damage of both small-and mid-sized pulmonary vessels. Two cases were complicated by invasive fungal disease consistent with Aspergillosis, and another by coinfection with cytomegalovirus. Our findings indicate that the lungs of patients who die of SARS are almost always positive for the SARS-associated coronavirus by RT-PCR, and may show features of both diffuse alveolar damage and acute fibrinous and organizing pneumonia patterns of acute injury. Cases of SARS may be complicated by coexistent infections and therapy-related lung injury.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>39f33cf580fe6ce842e2c9005570d08115c3ed4b</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Coronavirus Susceptibility to the Antiviral Remdesivir (GS- 5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Maria Agostini</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Vanderbilt University Medical Center</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Emerging coronaviruses (CoVs) cause severe disease in humans, but no approved therapeutics are available. The CoV nsp14 exoribonuclease (ExoN) has complicated development of antiviral nucleosides due to its proofreading activity. We recently reported that the nucleoside analogue GS-5734 (remdesivir) potently inhibits human and zoonotic CoVs in vitro and in a severe acute respiratory syndrome coronavirus (SARS-CoV) mouse model. However, studies with GS-5734 have not reported resistance associated with GS-5734, nor do we understand the action of GS-5734 in wild-type (WT) proofreading CoVs. Here, we show that GS-5734 inhibits murine hepatitis virus (MHV) with similar 50% effective concentration values (EC 50 ) as SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV). Passage of WT MHV in the presence of the GS-5734 parent nucleoside selected two mutations in the nsp12 polymerase at residues conserved across all CoVs that conferred up to 5.6-fold resistance to GS-5734, as determined by EC 50 . The resistant viruses were unable to compete with WT in direct coinfection passage in the absence of GS-5734. Introduction of the MHV resistance mutations into SARS-CoV resulted in the same in vitro resistance phenotype and attenuated SARS-CoV pathogenesis in a mouse model. Finally, we demonstrate that an MHV mutant lacking ExoN proofreading was significantly more sensitive to GS-5734. Combined, the results indicate that GS-5734 interferes with the nsp12 polymerase even in the setting of intact ExoN proofreading activity and that resistance can be overcome with increased, nontoxic concentrations of GS-5734, further supporting the development of GS-5734 as a broad-spectrum therapeutic to protect against contemporary and emerging CoVs. IMPORTANCE Coronaviruses (CoVs) cause severe human infections, but there are no approved antivirals to treat these infections. Development of nucleoside-based therapeutics for CoV infections has been hampered by the presence of a proofreading exoribonuclease. Here, we expand the known efficacy of the nucleotide prodrug remdesivir (GS-5734) to include a group ‚ê§-2a CoV. Further, GS-5734 potently inhibits CoVs with intact proofreading. Following selection with the GS-5734 parent nucleoside, 2 amino acid substitutions in the nsp12 polymerase at residues that are identical across CoVs provide low-level resistance to GS-5734. The resistance mutations decrease viral fitness of MHV in vitro and attenuate pathogenesis in a SARS-CoV animal model of infection. Together, these studies define the target of GS-5734 activity and demonstrate that resistance is difficult to select, only partial, and impairs fitness</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>3f4d5231b5f60eb3af06dfc1e7239238e2e0ddbd</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>False-Negative Results of Real-Time Reverse- Transcriptase Polymerase Chain Reaction for Severe Acute Respiratory Syndrome Coronavirus 2: Role of Deep-Learning-Based CT Diagnosis and Insights from Two Cases CASE REPORT Case 1 A 10-month-old boy presented with fever for 3 hours Case Report | Thoracic Imaging</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Dasheng Li</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.3348/kjr.2020.0146</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The epidemic of 2019 novel coronavirus, later named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is still gradually spreading worldwide. The nucleic acid test or genetic sequencing serves as the gold standard method for confirmation of infection, yet several recent studies have reported false-negative results of real-time reverse-transcriptase polymerase chain reaction (rRT-PCR). Here, we report two representative false-negative cases and discuss the supplementary role of clinical data with rRT-PCR, including laboratory examination results and computed tomography features. Coinfection with SARS-COV-2 and other viruses has been discussed as well.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>48106886ec5b19e6cc62abf552ff3529f1d8aca3</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>The presence of fever in adults with influenza and other viral respiratory infections</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>A Chughtai</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of New South Wales</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>We compared the rates of fever in adult subjects with laboratory-confirmed influenza and other respiratory viruses and examined the factors that predict fever in adults. Symptom data on 158 healthcare workers (HCWs) with a laboratory-confirmed respiratory virus infection were collected using standardized data collection forms from three separate studies. Overall, the rate of fever in confirmed viral respiratory infections in adult HCWs was 23¬∑4% (37/158). Rates varied by virus: human rhinovirus (25¬∑3%, 19/75), influenza A virus (30%, 3/10), coronavirus (28¬∑6%, 2/7), human metapneumovirus (28¬∑6%, 2/7), respiratory syncytial virus (14¬∑3%, 4/28) and parainfluenza virus (8¬∑3%, 1/12). Smoking [relative risk (RR) 4¬∑65, 95% confidence interval (CI) 1¬∑33-16¬∑25] and coinfection with two or more viruses (RR 4¬∑19, 95% CI 1¬∑21-14¬∑52) were significant predictors of fever. Fever is less common in adults with confirmed viral respiratory infections, including influenza, than described in children. More than 75% of adults with a viral respiratory infection do not have fever, which is an important finding for clinical triage of adult patients with respiratory infections. The accepted definition of 'influenza-like illness' includes fever and may be insensitive for surveillance when high case-finding is required. A more sensitive case definition could be used to identify adult cases, particularly in event of an emerging viral infection.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>58657873c0bc33b6cbd816292e2d239228cde3a0</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Metagenomic Nanopore Sequencing of Influenza Virus Direct from Clinical Respiratory Samples</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Kuiama Lewandowski</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Public Health England</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential for a diagnostic test for influenza virus which also provides insights on transmission, evolution, and drug resistance and simultaneously detects other viruses. We therefore set out to apply the Oxford Nanopore Technologies sequencing method to metagenomic sequencing of respiratory samples. We generated influenza virus reads down to a limit of detection of 10 2 to 10 3 genome copies/ml in pooled samples, observing a strong relationship between the viral titer and the proportion of influenza virus reads (P œ≠ 4.7 œ´ 10 œ™5 ). Applying our methods to clinical throat swabs, we generated influenza virus reads for 27/27 samples with mid-to-high viral titers (cycle threshold [C T ] values, œΩ30) and 6/13 samples with low viral titers (C T values, 30 to 40). No false-positive reads were generated from 10 influenza virus-negative samples. Thus, Nanopore sequencing operated with 83% sensitivity (95% confidence interval [CI], 67 to 93%) and 100% specificity (95% CI, 69 to 100%) compared to the current diagnostic standard. Coverage of full-length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza virus titers, we were able to reconstruct œæ99% complete sequences for all eight gene segments. We also detected a human coronavirus coinfection in one clinical sample. While further optimization is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza virus and other respiratory viruses.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>82aec91aec3367c18a7736eeb3e4a76839042b89</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Coinfection of pigs with porcine respiratory coronavirus and Bordetella bronchiseptica</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>S Brockmeier</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>National Animal Disease Center</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Coinfection with two or more pathogens is a common occurrence in respiratory diseases of most species. The manner in which multiple pathogens interact is not always straightforward, however. Bordetella bronchiseptica and porcine respiratory coronavirus (PRCV) are respiratory pathogens of pigs whose relatives, B. pertussis and the SARS virus, cause respiratory disease in humans. In an initial experiment, the effect of coinfection of PRCV and B. bronchiseptica was examined in thirty, 4week-old pigs (10 pigs/group) that were infected with either PRCV or B. bronchiseptica, or both PRCV and B. bronchiseptica. An additional 10 pigs served as sham infected controls. Five pigs from each group were euthanized at 4 and 10 days postinfection. Gross and histopathological lung lesions were more severe in the coinfected group as compared to the groups infected with B. bronchiseptica or PRCV alone. In order to investigate the potential role of proinflammatory cytokines in disease severity after coinfection, a second experiment was performed to examine cytokine transcription in alveolar macrophages from single and dually infected pigs. A total of 48 pigs were divided equally into groups as above, but 4 pigs from each group were euthanized at 1, 4 and 10 days post-infection. Coinfected pigs showed a greater and more sustained transcription of proinflammatory cytokines, especially IL-6 and MCP-1, than pigs infected with either PRCV or B. bronchiseptica alone.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>857573da5352641c02c784b1e19d92fb2e49e70b</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Enhanced detection of respiratory pathogens with nanotrap particles</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Nazly Shafagati</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>George Mason University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The Influenza virus is a leading cause of respiratory disease in the United States each year. While the virus normally causes mild to moderate disease, hospitalization and death can occur in many cases. There are several methodologies that are used for detection; however problems such as decreased sensitivity and high rates of false-negative results may arise. There is a crucial need for an effective sample preparation technology that concentrates viruses at low abundance while excluding resident analytes that may interfere with detection. Nanotrap particles are hydrogel particles that are coupled to chemical dye affinity baits that bind a broad range of proteins and virions. Within minutes (&lt;30 minutes), Nanotrap particles concentrate low abundant proteins and viruses from clinically complex matrices. Nanotrap particles with reactive red baits concentrated numerous respiratory viruses including various strains and subtypes of Influenza virus, Coronavirus, and Respiratory Syncytial Virus from saliva, nasal fluid swab specimens, and nasal aspirates. Detection was enhanced more than 10-fold when coupled to plaque assays and qRT-PCR. Importantly, Nanotrap particle can efficiently capture and concentrate multiple viral pathogens during a coinfection scenario. These results collectively demonstrate that Nanotrap particles are an important tool that can easily be integrated into various detection methodologies.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>8e8bfa3e7f71d14244026b036057d5dd2d2e8647</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Prevalence and risk factors for agents causing diarrhea (Coronavirus, Rotavirus, Cryptosporidium spp., Eimeria spp., and nematodes helminthes) according to age in dairy calves from Brazil</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Leonardo Bueno Cruvinel</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The present study attempted to verify the prevalence of and risk factors for diarrhea-causing agents in dairy calves from Brazil. Additionally, ages with a higher risk of occurrence for each agent were verified by means of the receiver operating characteristic (ROC) curve. The collections were performed on 39 farms, belonging to 29 municipalities located in eight states of Brazil. It was possible to conclude that the prevalence of Coronavirus, Rotavirus, Cryptosporidium spp., Eimeria spp., and nematodes was 7.20% (95% CI 4.54-9.78), 6.37% (95% CI 3.85-8.89), 51.52% (95% CI 45.26-55.57), 3.46% (95% CI 2.24-4.67), and 3.46% (95% CI 2.24-4.67), respectively. Ages with higher probabilities of occurrence of these diseases in calves were &lt; 10, &gt; 8, &gt; 6, &gt; 37, and &gt; 36 days, respectively. Diarrhea occurred more significantly (P &lt; 0.0001) in animals less than 21 days old and mainly on those receiving milk through automatic feeders (P &lt; 0.001). Cryptosporidium spp. were a risk factor for the occurrence of Rotavirus, and vice versa (P = 0.0039) and presented a positive correlation with Coronavirus (P = 0.0089). Calves that drink water from rivers, streams, and ponds had a higher chance of being infected by Eimeria spp. (P &lt; 0.0001), as well as developing infection by nematodes (P &lt; 0.0001). The results found in this study highlight the importance of studying the agents of diarrhea together, once they act as coinfection where the losses triggered for the owners will involve some of these agents simultaneously.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>928c0266f9fede929216a7c6f3370d32ff638401</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Virological and clinical characterizations of respiratory infections in hospitalized children</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Suat Bicer</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: The purpose of this study was to determine the incidence and seasonal distribution of viral etiological agents and to compare their clinical manifestations and disease severity, including single and co infections. Methods: Multiplex reverse-transcription PCR was performed for the detection of viruses in nasopharyngeal aspirat. Disease severity was grouped using a categorization index as very mild/mild, and moderate/severe. Clinical and laboratory characteristics of hospitalized children with viral respiratory tract infection were analyzed. Results: Viral pathogens were detected in 103/155 (66.5%) of patients. In order of frequency, identified pathogens were respiratory syncytial virus (32.0%), adenovirus (26.2%), parainfluenza viruses type 1-4 (19.4%), rhinovirus (18.4%), influenza A and B (12.6%), human metapneumovirus (12.6%), coronavirus (2.9%), and bocavirus (0.9%). Coinfections were present in 21 samples. Most of the children had very mild (38.8%) and mild disease (37.9%). Severity of illness was not worse with coinfections. The most common discharge diagnoses were "URTI" with or without LRTI/asthma (n=58). Most viruses exhibited strong seasonal patterns. Leukocytosis (22.2%) and neutrophilia (36.6%) were most commonly detected in patients with adenovirus and rhinovirus (p&lt;0.05). Monocytosis was the most remarkable finding in the patients (n=48, 53.3%), especially in patients with adenovirus (p&lt;0.05).</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>a8cec4aaa782514ef42b320b845e9d2d1804b42d</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Respiratory viral infections are underdiagnosed in patients with suspected sepsis</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>L Ljungstr√∂m</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The study aim was to investigate the prevalence and clinical relevance of viral findings by multiplex PCR from the nasopharynx of clinically septic patients during a winter season. During 11 weeks of the influenza epidemic period in January-March 2012, consecutive adult patients suspected to be septic (n = 432) were analyzed with cultures from blood and nasopharynx plus multiplex PCR for respiratory viruses on the nasopharyngeal specimen. The results were compared with those from microbiology analyses ordered as part of standard care. During the winter season, viral respiratory pathogens, mainly influenza A virus, human metapneumovirus, coronavirus, and respiratory syncytial virus were clinically underdiagnosed in 70% of patients positive by the multiplex PCR assay. During the first four weeks of the influenza epidemic, few tests for influenza were ordered by clinicians, indicating low awareness that the epidemic had started. Nasopharyngeal findings of Streptococcus pneumoniae and Haemophilus influenzae by culture correlated to pneumonia diagnosis, and in those patients laboratory signs of viral coinfections were common but rarely suspected by clinicians. The role of respiratory viral infections in patients presenting with a clinical picture of sepsis is underestimated. Specific antiviral treatment might be beneficial in some cases and may reduce spread in a hospital setting. Diagnosing viral infections may promote reduction of unnecessary antibiotic use. It can also be a tool for decisions concerning patient logistics, in order to minimize exposure of susceptible patients and personnel.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>d95aa29fa61c9fd4ee5f7bc29fca44d9e124c517</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Epidemiology and microbiological investigations of community-acquired pneumonia in children admitted at the emergency department of a university hospital</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Aymeric Cantais</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>CHU de Saint-Etienne</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: The management of children with community-acquired pneumonia (CAP) is largely influenced by the development of new molecular diagnostic tests that allow the simultaneous detection of a wide range of pathogens. Objectives: Evaluation of a diagnostic approach including multiplex PCR assays for revisiting the epidemiology and etiology of CAP in children at hospital. Study design: Children of all ages consulting at the Emergency Department of the University hospital of Saint-Etienne, France, during the 2012-2013 winter period were included. In addition to bacterial cultures, the following pathogens were detected using biplex commercially-available rt-PCR tests: adenovirus, respiratory syncytial virus, human metapneumovirus, bocavirus, rhinovirus/enterovirus, coronavirus, influenza viruses A and B, parainfluenza viruses, Mycoplasma pneumoniae and Chlamydophila pneumonia. Results: From 85 patients with CAP, at least one pathogen was identified in 81 cases (95.3%), including 4 bacterial exclusive infections (4.7%), 53 viral exclusive infections (62.4%) and 24 mixed infections (28.2%). Coinfection by at least two viruses was observed in 37 cases (43.5%). Mean age was higher in the case of documented bacterial infection (P &lt; 0.05). In the subgroup of viral exclusive infection, the mean age of severe cases was 2.0 years vs 3.8 years in mild and moderate cases (P &lt; 0.05). Conclusions: These findings highlight the huge proportion of CAP of viral origin, the high number of coinfection by multiple viruses and the low number of bacterial CAP, notably in children under 5 years, and address the need to re-evaluate the indications of empiric antimicrobial treatment in this age group.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>ddf124684188ac29dba3b174f327bd0fd62155bd</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Co-Circulation of the Rare CPV-2c with Unique Gln370Arg Substitution, New CPV-2b with Unique Thr440Ala Substitution, and New CPV-2a with High Prevalence and Variation in Heilongjiang Province, Northeast China</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Yufei Geng</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Heilongjiang Bayi Agricultural University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>To trace evolution of canine parvovirus-2 (CPV-2), a total of 201 stool samples were collected from dogs with diarrhea in Heilongjiang province of northeast China from May 2014 to April 2015. The presence of CPV-2 in the samples was determined by PCR amplification of the VP2 gene (568 bp) of CPV-2. The results revealed that 95 samples (47.26%) were positive for CPV-2, and they showed 98.8%-100% nucleotide identity and 97.6%-100% amino acid identity. Of 95 CPV-2-positive samples, types new2a (Ser297Ala), new2b (Ser297Ala), and 2c accounted for 64.21%, 21.05%, and 14.74%, respectively. The positive rate of CPV-2 and the distribution of the new2a, new2b and 2c types exhibited differences among regions, seasons, and ages. Immunized dogs accounted for 48.42% of 95 CPV-2-positive samples. Coinfections with canine coronavirus, canine kobuvirus, and canine bocavirus were identified. Phylogenetic analysis revealed that the identified new2a, new2b, and CPV-2c strains in our study exhibited a close relationship with most of the CPV-2 strains from China; type new2a strains exhibited high variability, forming three subgroups; type new2b and CPV-2c strains formed one group with reference strains from China. Of 95 CPV-2 strains, Tyr324Ile and Thr440Ala substitutions accounted for 100% and 64.21%, respectively; all type new2b strains exhibited the Thr440Ala substitution, while the unique Gln370Arg substitution was found in all type 2c strains. Recombination analysis using entire VP2 gene indicated possible recombination events between the identified CPV-2 strains and reference strains from China. Our data revealed the co-circulation of new CPV-2a, new CPV-2b, and rare CPV-2c, as well as potential recombination events among Chinese CPV-2 strains.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>fd4ab143e7dd8db7255b11177505986e87b50943</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Increased Detection of Viruses in Children with Respiratory Tract Infection Using PCR</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Chien-Yu Lin</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>MacKay Memorial Hospital</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Respiratory viruses are a common cause of respiratory tract infection (RTI), particularly in neonates and children. Rapid and accurate diagnosis of viral infections could improve clinical outcomes and reduce the use of antibiotics and treatment sessions. Advances in diagnostic technology contribute to the accurate detection of viruses. We performed a multiplex real-time polymerase chain reaction (PCR) to investigate the viral etiology in pediatric patients and compared the detection rates with those determined using traditional antigen tests and virus cultures. Fifteen respiratory viruses were included in our investigation: respiratory syncytial virus A/B (RSV), influenza virus A (FluA) and influenza virus B (FluB), human metapneumovirus (MPV), enterovirus (EV), human parainfluenza virus (PIV) types 1-4, human rhinovirus (RV), human coronavirus OC43, NL63, and 229E, human adenovirus (ADV), and human bocavirus (Boca). In total, 474 specimens were collected and tested. Respiratory viruses were detected more frequently by PCR (357, 75.3%) than they were by traditional tests (229, 49.3%). The leading pathogens were RSV (113, 23.8%), RV (72, 15.2%), PIV3 (53, 11.2%), FluA (51, 10.8%), and ADV (48, 10.1%). For children younger than 5 years, RSV and RV were most prevalent; for children older than 5 years, FluA and ADV were the most frequently detected. Of the specimens, 25.8% (92/357) were coinfected with two or more viruses. RV, Boca, PIV2, FluB, and PIV4 had higher rates of coinfection; MPV and PIV1 had the lowest rates of coinfection (9.1% and 5.3%). To conclude, the detection power of PCR was better than that of traditional antigen tests and virus cultures when considering the detection of respiratory viruses. RSV and RV were the leading viral pathogens identified in the respiratory specimens. One-quarter of the positive specimens were coinfected with two or more viruses. In the future, further application of PCR may contribute to the rapid and accurate diagnosis of respiratory viruses and could improve patient outcomes.</td>
    </tr>
  </tbody>
</table>