<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>0</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>SHA</th>
      <td>2aff265697682af71816737a2fd85f1d9bc9a0c4</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Clinical Medicine The Rate of Underascertainment of Novel Coronavirus (2019-nCoV) Infection: Estimation Using Japanese Passengers Data on Evacuation Flights</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Hiroshi Nishiura</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hokkaido University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>From 29 to 31 January 2020, a total of 565 Japanese citizens were evacuated from Wuhan, China on three chartered flights. All passengers were screened upon arrival in Japan for symptoms consistent with novel coronavirus (2019-nCoV) infection and tested for presence of the virus. Assuming that the mean detection window of the virus can be informed by the mean serial interval (estimated at 7.5 days), the ascertainment rate of infection was estimated at 9.2% (95% confidence interval: 5.0, 20.0). This indicates that the incidence of infection in Wuhan can be estimated at 20,767 infected individuals, including those with asymptomatic and mildly symptomatic infections. The infection fatality risk (IFR)-the actual risk of death among all infected individuals-is therefore 0.3% to 0.6%, which may be comparable to Asian influenza pandemic of 1957-1958.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>5a64d8be518b46570d2567a704fc8d6d95bd1008</td>
    </tr>
    <tr>
      <th>Title</th>
      <td></td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Shi Zhao</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Chinese University of Hong Kong</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Backgrounds: An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city in China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and other countries. We present estimates of the basic reproduction number, R 0 , of 2019-nCoV in the early phase of the outbreak. Methods: Accounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (g), we estimated R 0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI. Findings: The early outbreak data largely follows the exponential growth. We estimated that the mean R 0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R 0 . Conclusion: The mean estimate of R 0 for the 2019-nCoV ranges from 2.24 to 3.58, and is significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>637939a0f462b9e821ff62fc20e9fedfe78df73e</td>
    </tr>
    <tr>
      <th>Title</th>
      <td></td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Shi Zhao</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Chinese University of Hong Kong</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Backgrounds: An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city in China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and other countries. We present estimates of the basic reproduction number, R 0 , of 2019-nCoV in the early phase of the outbreak. Methods: Accounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (g), we estimated R 0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI. Findings: The early outbreak data largely follows the exponential growth. We estimated that the mean R 0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R 0 . Conclusion: The mean estimate of R 0 for the 2019-nCoV ranges from 2.24 to 3.58, and is significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>89c1764db023518c70301bd34aa180d8c5d98e51</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>ECG Changes After Rabbit Coronavirus Infection</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Lorraine Alexander</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>This study examines the electrocardiographic (ECG) changes following rabbit coronavirus (RbCV) infection. We have shown that infection with RbCV results in the development of myocarditis and congestive heart failure and that some survivors of RbCV infection go on to develop dilated cardiomyopathy in the chronic phase. Serial ECGs were recorded on 31 RbCV-infected rabbits. Measurements of heart rate; P-R interval; QRS duration; QTc interval; and P-, QRS-, and T-wave voltages were taken. The recordings were also examined for disturbances of conduction, rhythm, and repolarization. The acute and subacute phases were characterized by sinus tachycardia with depressed R-and T-wave voltages as well as disturbances of conduction, rhythm, and repolarization. In most animals in the chronic phase, the sinus rate returned to near-baseline values with resolution of the QRS voltage changes. The ECG changes observed during RbCV infection are similar to the spectrum of interval/segment abnormalities, rhythm disturbances, conduction defects, and myocardial pathology seen in human myocarditis, heart failure, and dilated cardiomyopathy. Because animals often died suddenly in the absence of severe clinical signs of congestive heart failure during the acute phase, RbCV infection may increase ventricular vulnerability, resulting in sudden cardiac death. RbCV infection may provide a rare opportunity to study sudden cardiac death in an animal model in which the ventricle is capable of supporting veutricular fibrillation, and invasive techniques monitoring cardiac function can be performed.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Title: Serial interval of novel coronavirus (COVID-19) infections Correspondence to</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Hiroshi Nishiura</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hokkaido University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>1 Objective: To estimate the serial interval of novel coronavirus (COVID-19) 2 from information on 28 infector-infectee pairs. 3</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>a58220fe30325ce4b67376b18556722c2192c990</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Middle East Respiratory Syndrome Coronavirus Outbreak in the Republic of Korea, 2015 Korea Centers for Disease Control and Prevention*</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Objectives: The outbreak of Middle East respiratory syndrome coronavirus (MERS-CoV) infection in the Republic of Korea started from the index case who developed fever after returning from the Middle East. He infected 26 cases in Hospital C, and consecutive nosocomial transmission proceeded throughout the nation. We provide an epidemiologic description of the outbreak, as of July 2015. Methods: Epidemiological research was performed by direct interview of the confirmed patients and reviewing medical records. We also analyzed the incubation period, serial interval, the characteristics of superspreaders, and factors associated with mortality. Full genome sequence was obtained from sputum specimens of the index patient. Results: A total of 186 confirmed patients with MERS-CoV infection across 16 hospitals were identified in the Republic of Korea. Some 44.1% of the cases were patients exposed in hospitals, 32.8% were caregivers, and 13.4% were healthcare personnel. The most common presenting symptom was fever and chills. The estimated incubation period was 6.83 days and the serial interval was 12.5 days. A total of 83.2% of the transmission events were epidemiologically linked to five superspreaders, all of whom had pneumonia at presentation and contacted hundreds of people. Older age [odds ratio (OR) Z 4.86, 95% confidence interval (CI) 1.90e12.45] and underlying respiratory disease (OR Z 4.90, 95% CI 1.64 e14.65) were significantly associated with mortality. Phylogenetic analysis showed that the MERS-CoV of the index case clustered closest with a recent virus from Riyadh, Saudi Arabia. Conclusion: A single imported MERS-CoV infection case imposed a huge threat to public health and safety. This highlights the importance of robust preparedness and optimal infection prevention control. The lessons learned from the current outbreak will contribute to more up-to-date guidelines and global health security.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>bd6a6cd1c453c47af156f6e42ba837207ab45c63</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Middle East Respiratory Syndrome Coronavirus Outbreak in the Republic of Korea, 2015 Korea Centers for Disease Control and Prevention*</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Objectives: The outbreak of Middle East respiratory syndrome coronavirus (MERS-CoV) infection in the Republic of Korea started from the index case who developed fever after returning from the Middle East. He infected 26 cases in Hospital C, and consecutive nosocomial transmission proceeded throughout the nation. We provide an epidemiologic description of the outbreak, as of July 2015. Methods: Epidemiological research was performed by direct interview of the confirmed patients and reviewing medical records. We also analyzed the incubation period, serial interval, the characteristics of superspreaders, and factors associated with mortality. Full genome sequence was obtained from sputum specimens of the index patient. Results: A total of 186 confirmed patients with MERS-CoV infection across 16 hospitals were identified in the Republic of Korea. Some 44.1% of the cases were patients exposed in hospitals, 32.8% were caregivers, and 13.4% were healthcare personnel. The most common presenting symptom was fever and chills. The estimated incubation period was 6.83 days and the serial interval was 12.5 days. A total of 83.2% of the transmission events were epidemiologically linked to five superspreaders, all of whom had pneumonia at presentation and contacted hundreds of people. Older age [odds ratio (OR) Z 4.86, 95% confidence interval (CI) 1.90e12.45] and underlying respiratory disease (OR Z 4.90, 95% CI 1.64 e14.65) were significantly associated with mortality. Phylogenetic analysis showed that the MERS-CoV of the index case clustered closest with a recent virus from Riyadh, Saudi Arabia. Conclusion: A single imported MERS-CoV infection case imposed a huge threat to public health and safety. This highlights the importance of robust preparedness and optimal infection prevention control. The lessons learned from the current outbreak will contribute to more up-to-date guidelines and global health security.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>bf20dda99538a594eafc258553634fd9195104cb</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Clinical Medicine Estimating the Unreported Number of Novel Coronavirus (2019-nCoV) Cases in China in the First Half of January 2020: A Data-Driven Modelling Analysis of the Early Outbreak</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Shi Zhao</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Chinese University of Hong Kong</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: In December 2019, an outbreak of respiratory illness caused by a novel coronavirus (2019-nCoV) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of foreign countries. The 2019-nCoV cases might have been under-reported roughly from 1 to 15 January 2020, and thus we estimated the number of unreported cases and the basic reproduction number, R 0 , of 2019-nCoV. Methods: We modelled the epidemic curve of 2019-nCoV cases, in mainland China from 1 December 2019 to 24 January 2020 through the exponential growth. The number of unreported cases was determined by the maximum likelihood estimation. We used the serial intervals (SI) of infection caused by two other well-known coronaviruses (CoV), Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) CoVs, as approximations of the unknown SI for 2019-nCoV to estimate R 0 . Results: We confirmed that the initial growth phase followed an exponential growth pattern. The under-reporting was likely to have resulted in 469 (95% CI: 403-540) unreported cases from 1 to 15 January 2020. The reporting rate after 17 January 2020 was likely to have increased 21-fold (95% CI: 18-25) in comparison to the situation from 1 to 17 January 2020 on average. We estimated the R 0 of 2019-nCoV at 2.56 (95% CI: 2.49-2.63). Conclusion: The under-reporting was likely to have occurred during the first half of January 2020 and should be considered in future investigation.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>c8c052d27aaf8015316dcd2644fa5e0b3870cea1</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Corey Peak</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Harvard T.H. Chan School of Public Health</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101/2020.03.05.20031088</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Individual quarantine and active monitoring of contacts are core disease control strategies, particularly for emerging infectious diseases such as Coronavirus Disease 2019 . To estimate the comparative efficacy of these interventions to control COVID-19, we fit a stochastic branching model, comparing two sets of reported parameters for the dynamics of the disease. Our results suggest that individual quarantine may contain an outbreak of COVID-19 with a short serial interval (4.8 days) only in settings with high intervention performance where at least three-quarters of infected contacts are individually quarantined. However, in settings where this performance is unrealistically high and the outbreak of COVID-19 continues to grow, so too will the burden of the number of contacts traced for active monitoring or quarantine. In such circumstances where resources are prioritized for scalable interventions such as social distancing, we show active monitoring or individual quarantine of high-risk contacts can contribute synergistically to social distancing. To the extent that interventions based on contact tracing can be implemented, therefore, they can help mitigate the spread of COVID-19. Our model highlights the urgent need for more data on the serial interval and the extent of presymptomatic transmission in order to make data-driven policy decisions regarding the cost-benefit comparisons of individual quarantine vs. active monitoring of contacts.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>d4b843968ab01451c9553576eff63a52ad8ba9a4</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Journal Pre-proof Serial interval of novel coronavirus (COVID-19) infections Title: Serial interval of novel coronavirus (COVID-19) infections Authors</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Hiroshi Nishiura</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hokkaido University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Original article Highlights  The serial interval of novel coronavirus infections was estimated from a total of 28 infector-infectee pairs. J o u r n a l P r e -p r o o f -2 - The median serial interval is shorter than the median incubation period, suggesting a substantial proportion of pre-symptomatic transmission.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>e44b1713295bc4efbd758d8886b84fb68c725325</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Journal Pre-proof Serial interval of novel coronavirus (COVID-19) infections Title: Serial interval of novel coronavirus (COVID-19) infections Authors</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Hiroshi Nishiura</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hokkaido University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Original article Highlights  The serial interval of novel coronavirus infections was estimated from a total of 28 infector-infectee pairs. J o u r n a l P r e -p r o o f -2 - The median serial interval is shorter than the median incubation period, suggesting a substantial proportion of pre-symptomatic transmission.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>ef78166253de329621b80f8ea193609d0330f0b5</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Specific real-time reverse transcription-polymerase chain reaction for detection and quantitation of turkey coronavirus RNA in tissues and feces from turkeys infected with turkey coronavirus</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Yi-Ning Chen</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Purdue University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Turkey coronavirus (TCoV) infection causes acute atrophic enteritis in the turkey poults, leading to significant economic loss in the U.S. turkey industry. Rapid detection, differentiation, and quantitation of TCoV are critical to the diagnosis and control of the disease. A specific one-step real-time reverse transcriptionpolymerase chain reaction (RRT-PCR) assay for detection and quantitation of TCoV in the turkey tissues was developed using a dual-labeled fluorescent probe. The fluorogenic probe labeled with a reporter dye (FAM, 6-carboxytetramethylrhodamin) and a quencher dye (AbsoluteQuencher TM ) was designed to bind to a 186 base-pair fragment flanked by the two PCR primers targeting the 3 end of spike gene of TCoV. The assay was performed on different avian viruses and bacteria to determine the specificity as well as serial dilutions of TCoV for the sensitivity. Three animal trials were conducted to further validate the assay. Ten-day-old turkey poults were inoculated orally with 100 EID 50 of TCoV. Intestinal tissues (duodenum, jejunum, ileum, cecum), feces from the cloacal swabs, or feces from the floor were collected at 12 h, 1, 2, 3, 5, 7, and/or 14 days post-inoculation (DPI). RNA was extracted from each sample and subjected to the RRT-PCR. The designed primers and probe were specific for TCoV. Other non-TCoV avian viruses and bacteria were not amplified by RRT-PCR. The assay was highly sensitive and could quantitate between 10 2 and 10 10 copies/l of viral genome. The viral RNA in the intestine segments reached the highest level, 6 × 10 15 copies/l, in the jejunum at 5 DPI. Eighty-four intestine segments assayed by the developed RRT-PCR and immunofluorescence antibody assay (IFA) revealed that there were 6 segments negative for TCoV by both assays, 45 positive for TCoV by IFA, and 77 positive for TCoV by RRT-PCR. Turkey coronavirus was detected in the feces from the cloacal swabs or floor 1-14 DPI; however, the viral RNA load varied among different turkey poults at different intervals from different trials. The highest amount of viral RNA, 2.8 × 10 10 copies/l, in the feces was the one from the cloacal swab collected at 1 DPI. The average amount of TCoV RNA in the cloacal fecal samples was 10 times higher than that in the fecal droppings on the floor. Taken together, the results indicated that the developed RRT-PCR assay is rapid, sensitive, and specific for detection, differentiation, and quantitation of TCoV in the turkey tissues and should be helpful in monitoring the progression of TCoV induced acute enteritis in the turkey flocks.</td>
    </tr>
  </tbody>
</table>