<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>0</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>SHA</th>
      <td>2e18ad30b21c44e8b5e1f54fc015ce512c19e1a7</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Molecular aspects of MERS-CoV</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Ali Rabaan</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Johns Hopkins Aramco Healthcare</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Middle East respiratory syndrome coronavirus (MERS-CoV) is a betacoronavirus which can cause acute respiratory distress in humans and is associated with a relatively high mortality rate. Since it was first identified in a patient who died in a Jeddah hospital in 2012, the World Health Organization has been notified of 1735 laboratory-confirmed cases from 27 countries, including 628 deaths. Most cases have occurred in Saudi Arabia. MERS-CoV ancestors may be found in Old World bats of the Vespertilionidae family. After a proposed bat to camel switching event, transmission of MERS-CoV to humans is likely to have been the result of multiple zoonotic transfers from dromedary camels. Human-to-human transmission appears to require close contact with infected persons, with outbreaks mainly occurring in hospital environments. Outbreaks have been associated with inadequate infection prevention and control implementation, resulting in recommendations on basic and more advanced infection prevention and control measures by the World Health Organization, and issuing of government guidelines based on these recommendations in affected countries including Saudi Arabia. Evolutionary changes in the virus, particularly in the viral spike protein which mediates virus-host cell contact may potentially increase transmission of this virus. Efforts are on-going to identify specific evidence-based therapies or vaccines. The broad-spectrum antiviral nitazoxanide has been shown to have in vitro activity against MERS-CoV. Synthetic peptides and candidate vaccines based on regions of the spike protein have shown promise in rodent and non-human primate models. GLS-5300, a prophylactic DNA-plasmid vaccine encoding S protein, is the first MERS-CoV vaccine to be tested in humans, while monoclonal antibody, m336 has given promising results in animal models and has potential for use in outbreak situations.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>335c14ac989e0aef609bb1bd1e117fcdf662b4ea</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Isolation and identification of human coronavirus 229E from frequently touched environmental surfaces of a university classroom that is cleaned daily</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Tania Bonny</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Frequently touched surfaces of a university classroom that is cleaned daily contained viable human coronavirus 229E (CoV-229E). Tests of a CoV-229E laboratory strain under conditions that simulated the ambient light, temperature, and relative humidity conditions of the classroom revealed that some of the virus remained viable on various surfaces for 7 days, suggesting CoV-229E is relatively stable in the environment. Our findings reinforce the notion that contact transmission may be possible for this virus.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>3e5633a9a76a8c03e4ec57e4659872a63f045117</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Severe acute respiratory syndrome-related coronavirus: The species and its viruses -a statement of the Coronavirus Study Group</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Alexander Gorbalenya</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Lomonosov Moscow State University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101/2020.02.07.937862</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks. : bioRxiv preprint Defining novelty and the place of SARS-CoV-2 within the taxonomy of the Coronaviridae family During the 21 st century, researchers studying coronaviruses -a family of enveloped positivestranded RNA viruses of vertebrates 8 -were confronted several times with the question of coronavirus novelty, including two times when a severe or even life-threatening disease was introduced into humans from a zoonotic reservoir: this happened with severe acute respiratory</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>40c0532640a8bb9d9a0aac07d017d5982af33fdb</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Mutation of the S and 3c genes in genomes of feline coronaviruses</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Keisuke Oguma</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Nihon University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Feline coronavirus (FCoV) is classified into two biotypes based on its pathogenicity in cats: a feline enteric coronavirus of low pathogenicity and a highly virulent feline infectious peritonitis virus. It has been suspected that FCoV alters its biotype via mutations in the viral genome. The S and 3c genes of FCoV have been considered the candidates for viral pathogenicity conversion. In the present study, FCoVs were analyzed for the frequency and location of mutations in the S and 3c genes from faecal samples of cats in an animal shelter and the faeces, effusions, and tissues of cats that were referred to veterinary hospitals. Our results indicated that approximately 95% FCoVs in faeces did not carry mutations in the two genes. However, 80% FCoVs in effusion samples exhibited mutations in the S and 3c genes with remainder displaying a mutation in the S or 3c gene. It was also suggested that mutational analysis of the 3c gene could be useful for studying the horizontal transmission of FCoVs in multi-cat environments.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>58bb89f7722922bd9a47cd49e172c498d5cd73cf</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Middle East Respiratory Syndrome Coronavirus and the One Health concept</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Maged Gomaa Hemida</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>King Faisal University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is one of the major threats to the healthcare systems in some countries, especially in the Arabian Peninsula. MERS-CoV is considered an ideal example of the One Health concept. This is due to the animals, especially dromedary camels, play important roles in the transmission and sustainability of the virus, and the virus can be transmitted through aerosols of infected patients into the environment. However, there is some debate regarding the origin of MERS-CoV either from bats or other unknown reservoirs. The dromedary camel is the only identified animal reservoir to date. These animals play important roles in sustaining the virus in certain communities and may act as an amplifier of the virus by secreting it in their body fluids, especially in nasal and rectal discharges. MERS-CoV has been detected in the nasal and rectal secretions of infected camels, and MERS-CoV of this origin has full capacity to infect human airway epithelium in both in vitro and in vivo models. Other evidence confirms the direct transmission of MERS-CoV from camels to humans, though the role of camel meat and milk products has yet to be well studied. Human-to-human transmission is well documented through contact with an active infected patient or some silently infected persons. Furthermore, there are some significant risk factors of individuals in close contact with a positive MERS-CoV patient, including sleeping in the same patient room, removing patient waste (urine, stool, and sputum), and touching respiratory secretions from the index case. Outbreaks within family clusters have been reported, whereby some blood relative patients were infected through their wives in the same house were not infected. Some predisposing genetic factors favor MERS-CoV infection in some patients, which is worth investigating in the near future. The presence of other comorbidities may be another factor. Overall, there are many unknown/confirmed aspects of the virus/human/animal network. Here, the most recent advances in this context are discussed, and the possible reasons behind the emergence and sustainability of MERS-CoV in certain regions are presented. Identification of the exact mechanism of transmission of MERS-CoV from camels to humans and searching for new reservoir/s are of high priority. This will reduce the shedding of the virus into the environment, and thus the risk of human infection can be mitigated.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>5f10ce361cbe5faf856bf5e8b31aec84580eb79a</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Human Airway Epithelial Cell Culture to Identify New Respiratory Viruses: Coronavirus NL63 as a Model</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Bridget Banach</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Loyola University Chicago Stritch School of Medicine</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Propagation of new human respiratory virus pathogens in established cell lines is hampered by a lack of predictability regarding cell line permissivity and by availability of suitable antibody reagents to detect infection in cell lines that do not exhibit significant cytopathic effect. Recently, molecular methods have been used to amplify and identify novel nucleic acid sequences directly from clinical samples, but these methods may be hampered by the quantity of virus present in respiratory secretions at different time points following the onset of infection. Human airway epithelial (HAE) cultures, which effectively mimic the human bronchial environment, allow for cultivation of a wide variety of human respiratory viral pathogens. The goal of the experiments described here was to determine if propagation and identification of a human respiratory virus may be achieved through inoculation of HAE cultures followed by whole transcriptome amplification (WTA) and sequence analysis. To establish proof-of-principle human coronavirus NL63 (HCoV-NL63) was evaluated, and the first visualization of HCoV-NL63 virus by transmission electron microscopy (TEM) is reported. Initial propagation of human respiratory secretions onto HAE cultures followed by TEM and WTA of culture supernatant may be a useful approach for visualization and detection of new human respiratory pathogens that have eluded identification by traditional approaches.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>6341c582ec98c886d4b5f76ef4ac402f066a54f1</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Human airway epithelial cell culture to identify new respiratory viruses: Coronavirus NL63 as a model</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Bridget Banach</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Loyola University Chicago Stritch School of Medicine</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Propagation of new human respiratory virus pathogens in established cell lines is hampered by a lack of predictability regarding cell line permissivity and by availability of suitable antibody reagents to detect infection in cell lines that do not exhibit significant cytopathic effect. Recently, molecular methods have been used to amplify and identify novel nucleic acid sequences directly from clinical samples, but these methods may be hampered by the quantity of virus present in respiratory secretions at different time points following the onset of infection. Human airway epithelial (HAE) cultures, which effectively mimic the human bronchial environment, allow for cultivation of a wide variety of human respiratory viral pathogens. The goal of the experiments described here was to determine if propagation and identification of a human respiratory virus may be achieved through inoculation of HAE cultures followed by whole transcriptome amplification (WTA) and sequence analysis. To establish proof-of-principle human coronavirus NL63 (HCoV-NL63) was evaluated, and the first visualization of HCoV-NL63 virus by transmission electron microscopy (TEM) is reported. Initial propagation of human respiratory secretions onto HAE cultures followed by TEM and WTA of culture supernatant may be a useful approach for visualization and detection of new human respiratory pathogens that have eluded identification by traditional approaches.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>691da9d5576aa959e22114a1ebb35548efd276a0</td>
    </tr>
    <tr>
      <th>Title</th>
      <td></td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1101/2020.02.</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The Eyring's rate process theory and free volume concept, two very popular theories in chemistry and physics fields, are employed to treat infectious disease transmissions. The susceptible individuals are assumed to move stochastically from one place to another. The virus particle transmission rate is assumed to obey the Eyring's rate process theory and also controlled by how much free volume available in a system. The transmission process is considered to be a sequential chemical reaction, and the concentrations or fractions of four epidemiological compartments, the susceptible, the exposed, the infected, and the removed, can be derived and calculated. The obtained equations show that the basic reproduction number, R 0 , is not a constant, dependent on the volume fraction of virus particles, virus particle size, and virus particle packing structure, the energy barrier associated with susceptible individuals, and environment temperature. The developed models are applied to treat coronavirus disease 2019 (Covid-19) transmission and make predictions on peak time, peak infected, and R 0 . Our work provides a simple and straightforward approach to estimate how infection diseases evolve and how many people may be infected.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>79284efbde971538024ccbe888fa90bcd515d45c</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>The Effects of Temperature and Relative Humidity on the Viability of the SARS Coronavirus</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>K Chan</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Queen Mary Hospital</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The main route of transmission of SARS CoV infection is presumed to be respiratory droplets. However the virus is also detectable in other body fluids and excreta. The stability of the virus at different temperatures and relative humidity on smooth surfaces were studied. The dried virus on smooth surfaces retained its viability for over 5 days at temperatures of 22-25 • C and relative humidity of 40-50%, that is, typical air-conditioned environments. However, virus viability was rapidly lost (&gt;3 log 10 ) at higher temperatures and higher relative humidity (e.g., 38 • C, and relative humidity of &gt;95%). The better stability of SARS coronavirus at low temperature and low humidity environment may facilitate its transmission in community in subtropical area (such as Hong Kong) during the spring and in air-conditioned environments. It may also explain why some Asian countries in tropical area (such as Malaysia, Indonesia or Thailand) with high temperature and high relative humidity environment did not have major community outbreaks of SARS.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>9b6060a910dd95d43bf90727afb705e70665e12a</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Distribution of bat-borne viruses and environment patterns</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Aneta Afelt</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of Warsaw</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.1016/j.meegid.2017.12.009.,</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Environmental modifications are leading to biodiversity changes, loss and habitat disturbance. This in turn increases contacts between wildlife and hence the risk of transmission and emergence of zoonotic diseases. We analyzed the environment and land use using remote spatial data around the sampling locations of bats positive for coronavirus (21 sites) and astrovirus (11 sites) collected in 43 sites. A clear association between viruses and hosts was observed. Viruses associated to synanthropic bat genera, such as Myotis or Scotophilus were associated to highly transformed habitats with human presence while viruses associated to fruit bat genera were correlated with natural environments with dense forest, grassland areas and regions of high elevation. In particular, group C betacoronavirus were associated with mosaic habitats found in anthropized environments.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>af89f711e9ccd2f3d7156d7210a06aeac825763c</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Inactivation of surrogate coronaviruses on hard surfaces by health care germicides</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Rachel Hulkower</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: In the 2003 severe acute respiratory syndrome outbreak, finding viral nucleic acids on hospital surfaces suggested surfaces could play a role in spread in health care environments. Surface disinfection may interrupt transmission, but few data exist on the effectiveness of health care germicides against coronaviruses on surfaces. Methods: The efficacy of health care germicides against 2 surrogate coronaviruses, mouse hepatitis virus (MHV) and transmissible gastroenteritis virus (TGEV), was tested using the quantitative carrier method on stainless steel surfaces. Germicides were o-phenylphenol/ p-tertiary amylphenol) (a phenolic), 70% ethanol, 1:100 sodium hypochlorite, ortho-phthalaldehyde (OPA), instant hand sanitizer (62% ethanol), and hand sanitizing spray (71% ethanol). Results: After 1-minute contact time, for TGEV, there was a log 10 reduction factor of 3.2 for 70% ethanol, 2.0 for phenolic, 2.3 for OPA, 0.35 for 1:100 hypochlorite, 4.0 for 62% ethanol, and 3.5 for 71% ethanol. For MHV, log 10 reduction factors were 3.9 for 70% ethanol, 1.3 for phenolic, 1.7 for OPA, 0.62 for 1:100 hypochlorite, 2.7 for 62% ethanol, and 2.0 for 71% ethanol. Conclusion: Only ethanol reduced infectivity of the 2 coronaviruses by .3-log 10 after 1 minute. Germicides must be chosen carefully to ensure they are effective against viruses such as severe acute respiratory syndrome coronavirus.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>cfd0dbf05b77b0d303aaa222ae0f4e8d77d6f61b</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>MERS Coronavirus: An Emerging Zoonotic Virus</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Fang Li</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of Minnesota</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging virus that was first reported in humans in June 2012 [1] . To date, MERS-CoV continues to infect humans with a fatality rate of~35%. At least 27 countries have reported human infections with MERS-CoV (https://www.who.int/emergencies/mers-cov/en/). MERS-CoV is a zoonotic virus. Like severe acute respiratory syndrome coronavirus (SARS-CoV), MERS-CoV is believed to have originated from bats [2, 3] . However, whereas the bat-to-human transmission of SARS-CoV was likely mediated by palm civets as intermediate hosts, humans likely acquired MERS-CoV from dromedary camels [4] [5] [6] . Human-to-human transmission of MERS-CoV does occur, but it is limited mostly to health care environments [7, 8] . Moreover, whereas SARS-CoV recognizes angiotensin-converting enzyme 2 (ACE2) as a cellular receptor [9,10], MERS-CoV uses dipeptidyl peptidase 4 (DPP4) to enter target cells [11, 12] . Currently, no vaccines or antiviral therapeutics have been approved for the prevention or treatment of MERS-CoV infection, although a number of them have been developed preclinically and/or tested clinically [13] [14] [15] [16] .</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>d3070b290b020128e13189e64839240e22fcafa7</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Environmental sampling for respiratory pathogens in Jeddah airport during the 2013 Hajj season</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Ziad Memish</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Ministry of Health</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: Respiratory tract infections (RTIs) are common during the Hajj season and are caused by a variety of organisms, which can be transmitted via the air or contaminated surfaces. We conducted a study aimed at sampling the environment in the King Abdul Aziz International (KAAI) Airport, Pilgrims City, Jeddah, during Hajj season to detect respiratory pathogens. Methods: Active air sampling was conducted using air biosamplers, and swabs were used to sample frequently touched surfaces. A respiratory multiplex array was used to detect bacterial and viral respiratory pathogens. Results: Of the 58 environmental samples, 8 were positive for at least 1 pathogen. One air sample (1 of 18 samples, 5.5%) tested positive for influenza B virus. Of the 40 surface samples, 7 (17.5%) were positive for pathogens. These were human adenovirus (3 out of 7, 42.8%), human coronavirus OC43/HKU1 (3 out of 7, 42.8%), Haemophilus influenzae (1 out of 7, 14.2%), and Moraxella catarrhalis (1 out of 7, 14.2%). Chair handles were the most commonly contaminated surfaces. The handles of 1 chair were cocontaminated with coronavirus OC43/HKU1 and H influenzae. Conclusion: Respiratory pathogens were detected in the air and on surfaces in the KAAI Airport in Pilgrims City. Larger-scale studies based on our study are warranted to determine the role of the environment in transmission of respiratory pathogens during mass gathering events (eg, Hajj) such that public health preventative measures might be better targeted.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>db59c00ca304cff50531d66511fcf971f97a53ec</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Identification of pathogens and virulence profile of Rhodococcus equi and Escherichia coli strains obtained from sand of parks</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>M Fernandes</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Universidade Estadual "Júlio de Mesquita Filho"</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The identification of pathogens of viral (Rotavirus, Coronavirus), parasitic (Toxocara spp.) and bacterial (Escherichia coli, Salmonella spp., Rhodococcus equi) origin shed in feces, and the virulence profile of R. equi and E. coli isolates were investigated in 200 samples of sand obtained from 40 parks, located in central region of state of Sao Paulo, Brazil, using different diagnostic methods. From 200 samples analyzed, 23 (11.5%) strains of R. equi were isolated. None of the R. equi isolates showed a virulent (vapA gene) or intermediately virulent (vapB gene) profiles. Sixty-three (31.5%) strains of E. coli were identified. The following genes encoding virulence factors were identified in E. coli: eae, bfp, saa, iucD, papGI, sfa and hly. Phylogenetic classification showed that 63 E. coli isolates belonged to groups B1 (52.4%), A (25.4%) and B2 (22.2%). No E. coli serotype O157:H7 was identified. Eggs of Toxocara sp. were found in three parks and genetic material of bovine Coronavirus was identified in one sample of one park. No Salmonella spp. and Rotavirus isolates were identified in the samples of sand. The presence of R. equi, Toxocara sp, bovine Coronavirus and virulent E. coli isolates in the environment of parks indicates that the sanitary conditions of the sand should be improved in order to reduce the risks of fecal transmission of pathogens of zoonotic potential to humans in these places.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>e9c899d06f0e31a0d2c58cd4d82edb0086f9ed04</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>An outbreak of feline infectious peritonitis in a Taiwanese shelter: epidemiologic and molecular evidence for horizontal transmission of a novel type II feline coronavirus</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Ying-Ting Wang</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Feline infectious peritonitis (FIP) is a fatal disease caused by feline coronavirus (FCoV) infection. FCoV can be divided into serotypes I and II. The virus that causes FIP (FIPV) is believed to occur sporadically and spread infrequently from cat to cat. Recently, an FIP outbreak from an animal shelter was confirmed in Taiwan. FCoV from all the cats in this shelter were analyzed to determine the epidemiology of this outbreak. Thirteen of 46 (28.2%) cats with typical signs of FIP were identified. Among them, seven cats were confirmed by necropsy and/or histopathological examinations. Despite the fact that more than one FCoV was identified in this multi-cat environment, the eight FIP cats were invariably found to be infected with a type II FCoV. Sequence analysis revealed that the type II FIPV detected from fecal samples, body effusions and granulomatous tissue homogenates from the cats that succumbed to FIP all harbored an identical recombination site in their S gene. Two of the cats that succumbed to FIP were found to harbor an identical nonsense mutation in the 3c gene. Fecal shedding of this type II virus in the effusive form of FIP can be detected up to six days before death. Taken together, our data demonstrate that horizontal transmission of FIPV is possible and that FIP cats can pose a potential risk to other cats living in the same environment.</td>
    </tr>
  </tbody>
</table>