<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>0</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>SHA</th>
      <td>01b50fc23a12266022e2fdb79b3cd5fe317d3856</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Severe Acute Respiratory Syndrome (SARS) and the GDP. Part I : Epidemiology, virology, pathology and general health issues</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>R Li</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The health profession faces a new challenge with the emergence of a novel viral disease Severe Acute Respiratory Syndrome (SARS), a form of atypical pneumonia caused by a coronavirus termed SARS-CoV. This highly infectious disease has spread through 32 countries, infecting more than 8,400 patients with over 790 deaths in just over 6 months. Over one quarter of those infected were unsuspecting healthcare workers. The major transmission mode of SARS-coronavirus appears to be through droplet spread with other minor subsidiary modes of transmission such as close contact and fomites although air borne transmission has not been ruled out. There is as yet no definitive treatment protocol. Although the peak period of the outbreak is likely to have passed and the risk of SARS in the UK is therefore assessed to be low, the World Health Organisation has asked all countries to remain vigilant lest SARS re-emerges. Recent laboratory acquired cases of SARS reported from Taiwan and Beijing, China are a testimony to this risk. Until reliable diagnostic tests, vaccine and medications are available, control of SARS outbreaks depends on close surveillance, early identification of index cases, quick isolation of carriers and effective infection control and public health measures.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>469ed0f00c09e2637351c9735c306f27acf3aace</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>First two months of the 2019 Coronavirus Disease (COVID-19) epidemic in China: real- time surveillance and evaluation with a second derivative model</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Xinguang Chen</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Background: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. Methods: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. Results: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. Conclusions: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>53633e0dad6b1d750f45e9ff280c78075267f731</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Epidemic Models of Contact Tracing: Systematic Review of Transmission Studies of Severe Acute Respiratory Syndrome and Middle East Respiratory Syndrome-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/)</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Kin Kwok</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hong Kong Special Administrative Region</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The emergence and reemergence of coronavirus epidemics sparked renewed concerns from global epidemiology researchers and public health administrators. Mathematical models that represented how contact tracing and follow-up may control Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) transmissions were developed for evaluating different infection control interventions, estimating likely number of infections as well as facilitating understanding of their likely epidemiology. We reviewed mathematical models for contact tracing and follow-up control measures of SARS and MERS transmission. Model characteristics, epidemiological parameters and intervention parameters used in the mathematical models from seven studies were summarized. A major concern identified in future epidemics is whether public health administrators can collect all the required data for building epidemiological models in a short period of time during the early phase of an outbreak. Also, currently available models do not explicitly model constrained resources. We urge for closed-loop communication between public health administrators and modelling researchers to come up with guidelines to delineate the collection of the required data in the midst of an outbreak and the inclusion of additional logistical details in future similar models.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>6abb30ae61aa5e41f16a28b9437940d5d76d745b</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Cell Discovery Phase-adjusted estimation of the number of Coronavirus Disease 2019 cases in Wuhan, China</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Huwen Wang</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV/SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R 0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious, and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of R t . In the first assumption, R t was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with R t = 1.9, 2.6, or 3.1. The number of infections would reach 11,044, 70,258, and 227,989, respectively, by 29 February 2020. In the second assumption, R t was assumed to gradually decrease at different phases from high level of transmission (R t = 3.1, 2.6, and 1.9) to below 1 (R t = 0.9 or 0.5) owing to increasingly implemented public health intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce R t to an ideal level and control the infection.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>7e6a3be51808891b99a73f57faaa454670055e45</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Epidemic Models of Contact Tracing: Systematic Review of Transmission Studies of Severe Acute Respiratory Syndrome and Middle East Respiratory Syndrome-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/)</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Kin Kwok</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hong Kong Special Administrative Region</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The emergence and reemergence of coronavirus epidemics sparked renewed concerns from global epidemiology researchers and public health administrators. Mathematical models that represented how contact tracing and follow-up may control Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) transmissions were developed for evaluating different infection control interventions, estimating likely number of infections as well as facilitating understanding of their likely epidemiology. We reviewed mathematical models for contact tracing and follow-up control measures of SARS and MERS transmission. Model characteristics, epidemiological parameters and intervention parameters used in the mathematical models from seven studies were summarized. A major concern identified in future epidemics is whether public health administrators can collect all the required data for building epidemiological models in a short period of time during the early phase of an outbreak. Also, currently available models do not explicitly model constrained resources. We urge for closed-loop communication between public health administrators and modelling researchers to come up with guidelines to delineate the collection of the required data in the midst of an outbreak and the inclusion of additional logistical details in future similar models.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>7e84a68e753fa2734beff117c24743d13b021c82</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Spatiotemporal Clustering of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Incidence in Saudi Arabia, 2012-2019</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Khalid Al-Ahmadi</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Middle East respiratory syndrome coronavirus (MERS-CoV) is a great public health concern globally. Although 83% of the globally confirmed cases have emerged in Saudi Arabia, the spatiotemporal clustering of MERS-CoV incidence has not been investigated. This study analysed the spatiotemporal patterns and clusters of laboratory-confirmed MERS-CoV cases reported in Saudi Arabia between June 2012 and March 2019. Temporal, seasonal, spatial and spatiotemporal cluster analyses were performed using Kulldorff's spatial scan statistics to determine the time period and geographical areas with the highest MERS-CoV infection risk. A strongly significant temporal cluster for MERS-CoV infection risk was identified between April 5 and May 24, 2014. Most MERS-CoV infections occurred during the spring season (41.88%), with April and May showing significant seasonal clusters. Wadi Addawasir showed a high-risk spatial cluster for MERS-CoV infection. The most likely high-risk MERS-CoV annual spatiotemporal clusters were identified for a group of cities (n = 10) in Riyadh province between 2014 and 2016. A monthly spatiotemporal cluster included Jeddah, Makkah and Taif cities, with the most likely high-risk MERS-CoV infection cluster occurring between April and May 2014. Significant spatiotemporal clusters of MERS-CoV incidence were identified in Saudi Arabia. The findings are relevant to control the spread of the disease. This study provides preliminary risk assessments for the further investigation of the environmental risk factors associated with MERS-CoV clusters. these included persons who handle live camels (e.g., camel workers, camel salesmen, camel shepherds, slaughterhouse workers, animal and waste transporters) and persons who administer medications to camels or clean camel medical equipment [6] [7] [8] . Nevertheless, most patients report no exposure to camels [9] . The global camel population is estimated at approximately 35 million camels [10]; of these, 95% are dromedary camels [11] . In Saudi Arabia, the camel is considered a national, socio-cultural heritage animal and is central in the livestock economy, as it is used for milk or meat and in a traditional race festival (locally known as Mazain al-Ibal) [12] . The number of camels in Saudi Arabia is estimated at more than 1.39 million heads [13] and has been growing annually by 5.2% since 1961 [10]. Information is lacking regarding the distribution of camels in Saudi Arabia. In June 2019, the Saudi Ministry of Environment, Water and Agriculture launched an electronic system for numbering and tracking camels in order to create a database on them through the use of microchip implants [14] . This endeavor is a part of a wider initiative to control animal diseases and to create a database on each camel and its owner. The new project will help improve camel health and control the spread of zoonotic diseases.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>7ee83287692dd2e39ce626028890e121639abbfc</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Evaluating the 3C-like protease activity of SARS-Coronavirus: Recommendations for standardized assays for drug discovery</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Valerie Grum-Tokars</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of Illinois</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Although the initial outbreaks of the deadly coronavirus that causes severe acute respiratory syndrome (SARS-CoV) were controlled by public health measures, the development of vaccines and antiviral agents for SARS-CoV is essential for improving control and treatment of future outbreaks. One potential target for SARS-CoV antiviral drug development is the 3C-like protease (3CLpro). This enzyme is an attractive target since it is essential for viral replication, and since there are now a number of high resolution X-ray structures of SARS-CoV 3CLpro available making structure-based drug-design possible. As a result, SARS-CoV 3CLpro has become the focus of numerous drug discovery efforts worldwide, but as a consequence, a variety of different 3CLpro expression constructs and kinetic assays have been independently developed making evaluation and comparison between potential inhibitors problematic. Here, we review the literature focusing on different SARS-CoV 3CLpro expression constructs and assays used to measure enzymatic activity. Moreover, we provide experimental evidence showing that the activity of 3CLpro enzymatic is significantly reduced when non-native sequences or affinity-tags are added to the N-or C-termini of the enzyme, or when the enzyme used in assays is at concentrations below the equilibrium dissociation constant of the 3CLpro dimer. We demonstrate for the first time the utility of a highly sensitive and novel Alexa488-QSY7 FRET-based peptide substrate designed for routine analysis and high-throughput screening, and show that kinetic constants determined from FRET-based assays that are uncorrected for inner-filter effects can lead to artifacts. Finally, we evaluated the effects of common assay components including DTT, NaCl, EDTA and DMSO on enzymatic activity, and we recommend standardized assay conditions and constructs for routine SARS-CoV 3CLpro assays to facilitate direct comparisons between SARS-CoV 3CLpro inhibitors under development worldwide.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>977370a5432431d0680323ce3f9e67db65f5f7b1</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Origin, Evolution, and Genotyping of Emergent Porcine Epidemic Diarrhea Virus Strains in the United States</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Yao-Wei Huang</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Zhejiang University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Coronaviruses are known to infect humans and other animals and cause respiratory and gastrointestinal diseases. Here we report the emergence of porcine epidemic diarrhea virus (PEDV) in the United States and determination of its origin, evolution, and genotypes based on temporal and geographical evidence. Histological lesions in small intestine sections of affected pigs and the complete genomic sequences of three emergent strains of PEDV isolated from outbreaks in Minnesota and Iowa were characterized. Genetic and phylogenetic analyses of the three U.S. strains revealed a close relationship with Chinese PEDV strains and their likely Chinese origin. The U.S. PEDV strains underwent evolutionary divergence, which can be classified into two sublineages. The three emergent U.S. strains are most closely related to a strain isolated in 2012 from Anhui Province in China, which might be the result of multiple recombination events between different genetic lineages or sublineages of PEDV. Molecular clock analysis of the divergent time based on the complete genomic sequences is consistent with the actual time difference, approximately 2 to 3 years, of the PED outbreaks between China (December 2010) and the United States (May 2013). The finding that the emergent U.S. PEDV strains share unique genetic features at the 5=-untranslated region with a bat coronavirus provided further support of the evolutionary origin of PEDV from bats and potential cross-species transmission. The data from this study have important implications for understanding the ongoing PEDV outbreaks in the United States and will guide future efforts to develop effective preventive and control measures against PEDV. IMPORTANCE The sudden emergence of porcine epidemic diarrhea virus (PEDV), a coronavirus, for the first time in the United States causes significant economic and public health concerns. Since its recognition in May 2013, PEDV has rapidly spread across the United States, resulting in high mortality in piglets in more than 17 States now. The ongoing outbreaks of Middle East respiratory syndrome coronavirus in humans from countries in or near the Arabian Peninsula and the historical deadly nature of the 2002 outbreaks of severe acute respiratory syndrome coronavirus create further anxiety over the emergence of PEDV in the United States due to the lack of scientific information about the origin and evolution of this emerging coronavirus. Here we report the detailed genetic characterization, origin, and evolution of emergent PEDV strains in the United States. The results provide much needed information to devise effective preventive and control strategies against PEDV in the United States. Citation Huang YW, Dickerman AW, Piñeyro P, Li L, Fang L, Kiehne R, Opriessnig T, Meng XJ. 2013. Origin, evolution, and genotyping of emergent porcine epidemic diarrhea virus strains in the United States. mBio 4(5):00737-13.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>a67a570efab57b9f2492a652c0c17f47546416c4</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Reverse Genetics of SARS-Related Coronavirus Using Vaccinia Virus-Based Recombination</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Sjoerd Van Den Worm</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Leiden University Medical Center</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Severe acute respiratory syndrome (SARS) is a zoonotic disease caused by SARS-related coronavirus (SARS-CoV) that emerged in 2002 to become a global health concern. Although the original outbreak was controlled by classical public health measures, there is a real risk that another SARS-CoV could re-emerge from its natural reservoir, either in its original form or as a more virulent or pathogenic strain; in which case, the virus would be difficult to control in the absence of any effective antiviral drugs or vaccines. Using the well-studied SARS-CoV isolate HKU-39849, we developed a vaccinia virusbased SARS-CoV reverse genetic system that is both robust and biosafe. The SARS-CoV genome was cloned in separate vaccinia virus vectors, (vSARS-CoV-5prime and vSARS-CoV-3prime) as two cDNAs that were subsequently ligated to create a genome-length SARS-CoV cDNA template for in vitro transcription of SARS-CoV infectious RNA transcripts. Transfection of the RNA transcripts into permissive cells led to the recovery of infectious virus (recSARS-CoV). Characterization of the plaques produced by recSARS-CoV showed that they were similar in size to the parental SARS-CoV isolate HKU-39849 but smaller than the SARS-CoV isolate Frankfurt-1. Comparative analysis of replication kinetics showed that the kinetics of recSARS-CoV replication are similar to those of SARS-CoV Frankfurt-1, although the titers of virus released into the culture supernatant are approximately 10-fold less. The reverse genetic system was finally used to generate a recSARS-CoV reporter virus expressing Renilla luciferase in order to facilitate the analysis of SARS-CoV gene expression in human dendritic cells (hDCs). In parallel, a Renilla luciferase gene was also inserted into the genome of human coronavirus 229E (HCoV-229E). Using this approach, we demonstrate that, in contrast to HCoV-229E, SARS-CoV is not able to mediate efficient heterologous gene expression in hDCs.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>badbdc6b3c6da8edee24cc4147009d6cbce08fd2</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Severe Acute Respiratory Syndrome (SARS) GEOGRAPHIC DISTRIBUTION AND MAGNITUDE OF DISEASE BURDEN Background</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Eileen Schneider</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Severe acute respiratory syndrome (SARS) first appeared in November 2002 and ultimately resulted in 8096 probable human infections and 774 deaths worldwide. By July 2003, the global outbreak was declared over. A new coronavirus, SARSassociated coronavirus (SARS-CoV), was identified as the causative agent; this virus appeared to have a zoonotic origin, as genetically similar coronaviruses have been identified in several animal species. The global response to the outbreak was extensive. Within a short period, the pathogen had been identified, new diagnostic tests were developed, surveillance systems were created, infection control and prevention measures were instituted, and transmission among humans stopped. It is unclear if and when person-to-person SARS-CoV transmission will reappear. However, procedures have been established by public health organizations, including the World Health Organization (WHO) and the Centers for Disease Control and Prevention (CDC), to help guide diagnosis, reporting, surveillance, and prevention.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>de2b71bc5b1434f5bd2021ce60cb9577f1e469b2</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Evaluating the 3C-like protease activity of SARS-Coronavirus: Recommendations for Standardized Assays for Drug Discovery NIH Public Access</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Valerie Grum-Tokars</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>University of Illinois</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>Although the initial outbreaks of the deadly coronavirus that causes severe acute respiratory syndrome (SARS-CoV) were controlled by public health measures, the development of vaccines and antiviral agents for SARS-CoV is essential for improving control and treatment of future outbreaks. One potential target for SARS-CoV antiviral drug development is the 3C-like protease (3CLpro). This enzyme is an attractive target since it is essential for viral replication, and since there are now a number of high resolution x-ray structures of SARS-CoV 3CLpro available making structure-based drug-design possible. As a result, SARS-CoV 3CLpro has become the focus of numerous drug discovery efforts worldwide, but as a consequence, a variety of different 3CLpro expression constructs and kinetic assays have been independently developed making evaluation and comparison between potential inhibitors problematic. Here, we review the literature focusing on different SARS-CoV 3CLpro expression constructs and assays used to measure enzymatic activity. Moreover, we provide experimental evidence showing that the activity of 3CLpro enzymatic is significantly reduced when non-native sequences or affinity-tags are added to the N-or C-termini of the enzyme, or when the enzyme used in assays is at concentrations below the equilibrium dissociation constant of the 3CLpro dimer. We demonstrate for the first time the utility of a highly sensitive and novel Alexa488-QSY7 FRET-based peptide substrate designed for routine analysis and high-throughput screening, and show that kinetic constants determined from FRET-based assays that are uncorrected for inner-filter effects can lead to artifacts. Finally, we evaluated the effects of common assay components including DTT, NaCl, EDTA and DMSO on enzymatic activity, and we recommend standardized assay conditions and constructs for routine SARS-CoV 3CLpro assays to facilitate direct comparisons between SARS-CoV 3CLpro inhibitors under development worldwide.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>e1b312167d5ea6a1ed3c071b05b7bd0a48a975df</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>Structural Factors of the Middle East Respiratory Syndrome Coronavirus Outbreak as a Public Health Crisis in Korea and Future Response Strategies</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Dong-Hyun Kim</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Hallym University College of Medicine</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The recent Middle East respiratory syndrome coronavirus (MERS-CoV) outbreak has originated from a failure in the national quarantine system in the Republic of Korea as most basic role of protecting the safety and lives of its citizens. Furthermore, a number of the Korean healthcare system's weaknesses seem to have been completely exposed. The MERS-CoV outbreak can be considered a typical public health crisis in that the public was not only greatly terrorized by the actual fear of the disease, but also experienced a great impact to their daily lives, all in a short period of time. Preparedness for and an appropriate response to a public health crisis require comprehensive systematic public healthcare measures to address risks comprehensively with an all-hazards approach. Consequently, discussion regarding establishment of post-MERS-CoV improvement measures must focus on the total reform of the national quarantine system and strengthening of the public health infrastructure. In addition, the Korea Centers for Disease Control and Prevention must implement specific strategies of action including taking on the role of "control tower" in a public health emergency, training of Field Epidemic Intelligence Service officers, establishment of collaborative governance between central and local governments for infection prevention and control, strengthening the roles and capabilities of community-based public hospitals, and development of nationwide crisis communication methods.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>ee6e4fcb9272a59d35f6ec53be5a8cff4d8c0b21</td>
    </tr>
    <tr>
      <th>Title</th>
      <td></td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Victor Corman¹</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. Aim: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. Methods: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. Results: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive -Global (EVAg), a European Union infrastructure project. Conclusion: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>fab32444594a7c25009ffcd48e91c923f5f0316f</td>
    </tr>
    <tr>
      <th>Title</th>
      <td>The imperative to develop a human vaccine for the Hendra virus in Australia</td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Bilal Zahoor</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Johns Hopkins University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>NA</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>The Hendra virus (HeV) poses a significant challenge to public health in Australia. Expanding migratory patterns observed among bats and the mutation of the virus to seek and successfully infect new hosts is a significant departure from the generalized epidemiological trend. The recent discovery of equine-related infections and deaths in addition to a canine infection demonstrates the inadequacy of the current equine vaccine developed in 2012. Traditional models for controlling the spread of the vector are futile given the rapid pace at which bats' habitats are eroded. Recent ongoing zoonotic epidemics, for example, Ebola and Middle East respiratory syndrome coronavirus, demonstrate that human-to-human transmission is a distinct reality rather than an obscure possibility. The development of a human HeV vaccine is essential for the biosecurity of Australia, as part of a multipronged strategy to control HeV in Australia.</td>
    </tr>
    <tr>
      <th>SHA</th>
      <td>fd28e6d03eef27b0454f13ca539dc1498242a4c2</td>
    </tr>
    <tr>
      <th>Title</th>
      <td></td>
    </tr>
    <tr>
      <th>First Author</th>
      <td>Ying-Hui Jin</td>
    </tr>
    <tr>
      <th>Institution</th>
      <td>Zhongnan Hospital of Wuhan University</td>
    </tr>
    <tr>
      <th>DOI</th>
      <td>10.</td>
    </tr>
    <tr>
      <th>Abstract</th>
      <td>In December 2019, a new type viral pneumonia cases occurred in Wuhan, Hubei Province; and then named "2019 novel coronavirus (2019-nCoV)" by the World Health Organization (WHO) on 12 January 2020. For it is a never been experienced respiratory disease before and with infection ability widely and quickly, it attracted the world's attention but without treatment and control manual. For the request from frontline clinicians and public health professionals of 2019-nCoV infected pneumonia management, an evidence-based guideline urgently needs to be developed. Therefore, we drafted this guideline according to the rapid advice guidelines methodology and general rules of WHO guideline development; we also added the first-hand management data of Zhongnan Hospital of Wuhan University. This guideline includes the guideline methodology, epidemiological characteristics, disease screening and population prevention, diagnosis, treatment and control (including traditional Chinese Medicine), nosocomial infection prevention and control, and disease nursing of the 2019-nCoV. Moreover, we also provide a whole process of a successful treatment case of the severe 2019-nCoV infected pneumonia and experience and lessons of hospital rescue for 2019-nCoV infections. This rapid advice guideline is suitable for the first frontline doctors and nurses, managers of hospitals and healthcare sections, community residents, public health persons, relevant researchers, and all person who are interested in the 2019-nCoV.</td>
    </tr>
  </tbody>
</table>